model-optimizer/extensions/middle/FusedBatchNormTraining.py

```diff
@@ -81,3 +82,14 @@ def replace_pattern(self, graph: Graph, match: dict):
                                        'override_output_shape': True}).create_node()
         reshape_back.in_port(1).connect(original_shape.out_port(0))
         mvn.out_port(0).get_connection().insert_node(reshape_back)
+
+        @staticmethod
+        def castBatchNormTraining2Inference(batchNormTrainingNode: Node) -> Node:
+            additional_attrs = {}
+            for batchNormAttr in  batchNormAttrList:
+                if batchNormTrainingNode.has(batchNormAttr):
+                    additional_attrs[batchNormAttr] = batchNormTrainingNode[batchNormAttr]
+            additional_attrs['name'] = batchNormTrainingNode.name
```

Comment:
Use `rename_nodes` function to rename original batchNormTraining node with batchNormInference node.